### DISCUSSION TITLE
v1.21.1

### DISCUSSION THREAD

**Discussion by heitorlessa** [aws-powertools/powertools-lambda-python#747] [Show and tell]
## Summary

Patch release to address regression in `Metrics` with mypy not recognizing a Callable when using `log_metrics()`.

### New Public Lambda Layers ARNs

Oh! It's finally here!!!

This release adds our first batch of public Lambda Layers for every AWS region supported by AWS Lambda - huge thanks to @am29d. 

This means you no longer need to deploy a SAR App in order to use Lambda Powertools as a Lambda Layer.

![image](https://user-images.githubusercontent.com/3340292/136413871-062b420a-a3bc-4b87-81bf-c1abda3e25c4.png)

That being said, we will keep SAR App in order to give you the flexibility to choose which semantic version you want to use as a Lambda Layer, until it is officially supported by Lambda Layers.

## 📜 Documentation updates

* docs: add new public layer ARNs (#746) by @am29d

## 🐛 Bug and hot fixes

* revert(metrics): typing regression on log_metrics callable (#744) by @heitorlessa

## 🔧 Maintenance

* chore(deps): bump boto3 from 1.18.54 to 1.18.56 (#742) by @dependabot
* chore(deps-dev): bump mkdocs-material from 7.3.1 to 7.3.2 (#741) by @dependabot
* chore: ignore constants in test cov (#745) by @heitorlessa

## This release was made possible by the following contributors:

@am29d, @dependabot, @dependabot[bot] and @heitorlessa

<hr /><em>This discussion was created from the release <a href='https://github.com/awslabs/aws-lambda-powertools-python/releases/tag/v1.21.1'>v1.21.1</a>.</em>

**Comment by mwarkentin** [comment]
@heitorlessa do you do any clean up of old layers or should they be safe to reference indefinitely? I know that was a bit of a painpoint with some other public layers we've used.

**Reply by heitorlessa** [reply]
We don't delete them, as we know it can break customers.

At some point we'll have to look into it more closely. By then, we should be able to see which layers are no longer being used and can be safely deleted.

cc @am29d

**Reply by am29d** [reply]
We will keep the layers forever (tm) and have no retention mechanism in place at the moment.
